hr_holidays_leave_overlap/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'HR - Holiday Leaves Overlap',
+    'summary': 'Handle leaves overlap',
+    'author': 'Onestein, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'website': 'http://www.onestein.eu',
```

Comment:
@espo-tony IMHO Change URL as per https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10

Reply:
@nikul-serpentcs your comment has been fulfilled, can you re-review?